lib/l10n/app_en.arb

```
@@ -2024,5 +2024,17 @@
   "unlinkTaskTitle": "Unlink Task",
   "viewMenuTitle": "View",
   "whatsNewDoneButton": "Done",
-  "whatsNewSkipButton": "Skip"
-}
\ No newline at end of file
+  "whatsNewSkipButton": "Skip",
+  "agentEvolutionVersionProposed": "Version proposed",
```

RuboCop (1.84.2)

[warning] Duplicated key in hash literal.

(Lint/DuplicateHashKey)